sway/input/seat.c

```diff
@@ -564,6 +564,12 @@ void seat_set_focus_warp(struct sway_seat *seat,
 		view_set_activated(view, false);
 	}
 
+	if (last_workspace && last_workspace != new_workspace) {
+		struct timespec now;
+		clock_gettime(CLOCK_MONOTONIC, &now);
+		cursor_send_pointer_motion(seat->cursor, now.tv_nsec / 1000);
```

Comment:
We do this in a lot of places. Maybe we should make zero a special value that gets the current time?

Reply:
Neat idea, I like it